openedx/features/specializations/templatetags/sso_meta_tag.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+from django.template.loader import get_template
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def sso_meta(context):
+    return get_template('features/specializations/sso_meta_template.html').render(context.flatten())
```

Comment:
how about using `inclusion_tag` instead of `simple_tag` ?

Reply:
https://docs.djangoproject.com/en/1.11/howto/custom-template-tags/#howto-custom-template-tags-inclusion-tags

Reply:
You are right and it is actually better but unfortunately we are using mako template and it can not be used here.